ShowerShow/Repository/Interface/IScheduleRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ShowerShow.DTO;
+using ShowerShow.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ShowerShow.Repository.Interface
+{
+    public interface IScheduleRepository
+    {
+        public Task CreateSchedule(CreateScheduleDTO schedule, Guid id);
+        public Task<List<Schedule>> GetAllSchedules(Guid UserId);
```

Comment:
Could you change List to IEnumerable ? It is a minor change but unfortunately they required it from us